src/Swagger/SchemaFormatter/SchemaFormatterProvider.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Swagger\SchemaFormatter;
+
+use ApiPlatform\Core\Exception\FormatterNotFoundException;
+
+class SchemaFormatterProvider
```

Comment:
For consistency with the rest of the code base (see DataProvider / DataPersister for instance), you should rename this class `ChainSchemaFormatter` and make it an implementation of `SchemaFormatterInterface`. Then only inject `SchemaFormatterInterface` in the normalizer.

Also:
```suggestion
final class SchemaFormatterProvider
```